lib/prelude.dx

```diff
@@ -2387,6 +2367,26 @@ def throw {a} (_:Unit) : {Except} a =
 def assert (b:Bool) : {Except} Unit =
   if not b then throw ()
 
+'### Misc instances that require `todo`
+
+instance {a b} Subset a (a|b)
+  inject' = \x. Left x
+  can_project' = \x. case x of
+    Left  x -> True
+    Right x -> False
+  unsafe_project' = \x. case x of
+    Left  x -> x
+    Right x -> todo   -- Impossible
```

Comment:
This situation is far from impossible: someone might call `unsafe_project` erroneously.

Reply:
Good point, this should be an `error`.